### PR TITLE
add give command

### PR DESCRIPTION
**Pull request by Joelrau**
here are some weapons codenames:
defaultweapon_mp
iw6_ak12_mp
iw6_ameli_mp
iw6_arx160_mp
iw6_bren_mp
iw6_cbjms_mp
iw6_fads_mp
iw6_fp6_mp
iw6_g28_mp
iw6_gm6_mp
iw6_gm6helisnipe_mp
iw6_honeybadger_mp
iw6_imbel_mp
iw6_k7_mp
iw6_kac_mp
iw6_knifeonly_mp
iw6_knifeonlyjugg_mp
iw6_kriss_mp
iw6_l115a3_mp
iw6_lsat_mp
iw6_m9a1_mp
iw6_m27_mp
iw6_maaws_mp
iw6_maawschild_mp
iw6_maawshoming_mp
iw6_magnum_mp
iw6_magnumhorde_mp
iw6_magnumjugg_mp
iw6_maul_mp
iw6_microtar_mp
iw6_minigun_mp
iw6_minigunjugg_mp
iw6_mk14_mp
iw6_mk32_mp
iw6_mk32horde_mp
iw6_mp443_mp
iw6_msbs_mp
iw6_mts255_mp
iw6_p226_mp
iw6_p226jugg_mp
iw6_panzerfaust3_mp
iw6_pdw_mp
iw6_pp19_mp
iw6_r5rgp_mp
iw6_rgm_mp
iw6_riotshield_mp
iw6_riotshieldjugg_mp
iw6_sc2010_mp
iw6_svu_mp
iw6_usr_mp
iw6_uts15_mp
iw6_vepr_mp
iw6_vks_mp

### CODE REVIEW

File: src/client/game/game.cpp [hunk 1]
```diff
@@ -230,6 +234,10 @@ namespace game
 			Dvar_Sort = Dvar_Sort_t(SELECT_VALUE(0x14042DEF0, 0x1404F1210));
 			Dvar_ValueToString = Dvar_ValueToString_t(SELECT_VALUE(0x14042E710, 0x1404F1A30));
 
+			G_GivePlayerWeapon = G_GivePlayerWeapon_t(SELECT_VALUE(0, 0x1403DA5E0));
```
Comment (momo5502) [inline src/client/game/game.cpp:237]: SP support would be nice

**Review by momo5502 — CHANGES_REQUESTED** [review]

File: src/client/game/game.cpp (hunk 1 above)
Comment (Joelrau) [reply src/client/game/game.cpp:237]: 👍 doing it

File: src/client/game/structs.hpp [hunk 2]
```diff
@@ -25,6 +25,18 @@ namespace game
 		unsigned int data;
 	};
 
+	enum errorParm
```
Comment (avail) [inline src/client/game/structs.hpp:28]: Parm?

**Review by momo5502 — CHANGES_REQUESTED** [review]
see comments in the commit

**Comment by avail** [comment]
> Also, yeah, parm?

actually I think that one is c&p from another place, iirc I saw errorParm in another PR some time ago

**Comment by momo5502** [comment]
> > Also, yeah, parm?
> 
> actually I think that one is c&p from another place, iirc I saw errorParm in another PR some time ago

little avail stalking IW6x pull requests :P

**Comment by avail** [comment]
> > > Also, yeah, parm?
> > 
> > 
> > actually I think that one is c&p from another place, iirc I saw errorParm in another PR some time ago
> 
> little avail stalking IW6x pull requests :P

heh I've watched this even before any code was in it :P

File: src/client/game/structs.hpp (hunk 2 above)
Comment (Joelrau) [reply src/client/game/structs.hpp:28]: > Also, yeah, parm?

That's what it's called in iw6_ds.pdb

**Comment by Joelrau** [comment]
> > Also, yeah, parm?
> 
> actually I think that one is c&p from another place, iirc I saw errorParm in another PR some time ago

yeah was from my fast_restart pr, but closed that one.

File: src/client/game/structs.hpp (hunk 2 above)
Comment (momo5502) [reply src/client/game/structs.hpp:28]: Haha fr? Wtf 😂

**Review by momo5502 — APPROVED** [review]

**Pull request merged**